src/atldld/cli.py

```diff
@@ -60,27 +62,35 @@ def cache_folder():
 
 
 # ============================= Dataset subcommand =============================
-
-
-@click.group(help="Commands related to atlas datasets")
-def dataset():
-    """Run dataset subcommands."""
-
-
-@click.command("info", help="Get information for a given dataset ID")
-@click.argument("dataset_id", type=int)
-def dataset_info(dataset_id):
-    """Get information for a given dataset ID."""
-    import textwrap
-
-    import atldld.dataset
+def get_dataset_meta_or_abort(
+    dataset_id: int, include: Optional[Sequence[str]] = None
+) -> Dict[str, Any]:
+    """Download the dataset metadata.
+
+    Parameters
+    ----------
+    dataset_id
+        The dataset ID.
+    include
+        The include keys to use in the RMA query.
+
+    Returns
+    -------
+    meta : dict
+        The dataset metadata.
+
+    Raises
+    ------
+    click.Abort
```

Comment:
Regarding the tracebacks, sure, users do not need to see them. But what about developers? Maybe we can have a `--debug` flag in the CLI? Or do this via logging?

Regarding the message, I would go for the `ClickException` since it will allow us to remove a lot of `capsys` logic in the tests.

Reply:
Right, replaced `click.Abort` by `click.ClickException` in a59e155

Reply:
Regarding tracebacks / debugging, there are for sure different options.

First of all, we already have a way because we added the `__main__.py` file. So now to debug some `atldld <command>` command one just needs to write `python -m pdb -m atldld <command>`. (I think one needs at least py37...)

And yes I've seen people using CLI flags to control the debug output. One could add something like `--debug`, `--log-level`, `-v`/`-vv`/`-vvv` directly to the root `atldld` command. For logging it would then configure the logging accordingly:
```python
import logging


logger = logging.getLogger(__name__)

@click.group()
@click.option('-v', '--verbose', count=True)
def root(verbose):
    """Run the command line interface for Atlas-Download-Tools."""
    if verbose == 0:
        log_level = logging.WARNING
    elif verbose == 1:
        log_level = logging.INFO
    else:  # verbose >= 2:
        log_level = logging.DEBUG
    logging.basicConfig(level=log_level)

    if verbose > 2:
        logger.warning("The maximal verbosity is -vv")


@root.command("test")
def test():
    logger.critical("Critical")
    logger.error("Error")
    logger.warning("Warning")
    logger.info("Info")
    logger.debug("Debug")
```

```bash
$ atldld test
CRITICAL:atldld.cli:Critical
ERROR:atldld.cli:Error
WARNING:atldld.cli:Warning
```

```bash
$ atldld -v test
CRITICAL:atldld.cli:Critical
ERROR:atldld.cli:Error
WARNING:atldld.cli:Warning
INFO:atldld.cli:Info
```

```bash
$ atldld -vv test
CRITICAL:atldld.cli:Critical
ERROR:atldld.cli:Error
WARNING:atldld.cli:Warning
INFO:atldld.cli:Info
DEBUG:atldld.cli:Debug
```

```bash
$ atldld -vvv test
WARNING:atldld.cli:The maximal verbosity is -vv
CRITICAL:atldld.cli:Critical
ERROR:atldld.cli:Error
WARNING:atldld.cli:Warning
INFO:atldld.cli:Info
DEBUG:atldld.cli:Debug
```

Reply:
By `debug` I actually meant more verbosity. Personally, I like the `-v -vv -vv` logic you proposed. However, it is also your call. If we decide to go for it I guess we can reuse this code in all CLI commands.

Reply:
Created #68